docs/contributing-to-airbyte/writing-docs.md

```diff
@@ -93,6 +96,68 @@ Please familiarize yourself with all the tools available to you when writing doc
+
+```md
+<Tabs>
+<TabItem value="http-basic" label="Basic HTTP authentication">
+
+When configuring this hypothetical connector using basic HTTP auth, you should mind some tricky security considerations! This just a hypothetical, though, so I never bothered to come up with any.
+
+As the first tab, this would be shown by default if no `TabItem` were marked as `default`.
+
+</TabItem>
+<TabItem value="oauth" label="OAuth authentication" default>
+
+When configuring this hypothetical connector using OAuth authentication, you should do a dance. Good for you! Since it's not the first `TabItem` in its set, we had to explicitly mark this tab as `default` for it to get top billing.
+
+- You don't need to mark any tab as `default`
+- If you don't, the first tab (here, Basic HTTP) will be the initial selection instead
+- You can use ordinary markdown syntax inside a `TabItem`
+- **however**, due to bugs in our in-app markdown rendering library, you should be dilligent about using empty lines to separate different formatting-related things (surrounding tags and their contents, paragraphs vs lists, etc)
+- You should also avoid indenting `TabItem` tags and their content according to html conventions, since text indented by four spaces (common for html nested inside two levels of tags) can be interpreted as a code block; different markdown rendering tools can handle this inconsistently.
+
+</TabItem>
+</Tabs>
+```
+
+That renders as the following:
+
+<Tabs>
+<TabItem value="http-basic" label="Basic HTTP authentication">
+
+When configuring this hypothetical connector using basic HTTP auth, you should mind some tricky security considerations! This just a hypothetical, though, so I never bothered to come up with any.
+
+As the first tab, this would be shown by default if no `TabItem` were marked as `default`.
+
+</TabItem>
+<TabItem value="oauth" label="OAuth authentication" default>
+
+When configuring this hypothetical connector using OAuth authentication, you should do a dance. Good for you! Since it's not the first `TabItem` in its set, we had to explicitly mark this tab as `default` for it to get top billing.
+
+- You don't need to mark any tab as `default`
+- If you don't, the first tab (here, Basic HTTP) will be the initial selection instead
+- You can use ordinary markdown syntax inside a `TabItem`
+- **however**, due to bugs in our in-app markdown rendering library, you should be dilligent about using empty lines to separate different formatting-related things (surrounding tags and their contents, paragraphs vs lists, etc)
+- You should also avoid indenting `TabItem` tags and their content according to html conventions, since text indented by four spaces (common for html nested inside two levels of tags) can be interpreted as a code block; different markdown rendering tools can handle this inconsistently.
+
+</TabItem>
+</Tabs>
```

Comment:
Currently this list is only shown when the second tab is selected. It should go outside of the `<Tabs>` tags, since it is relevant regardless of which tab is selected. 

```suggestion
</TabItem>
</Tabs>

- You don't need to mark any tab as `default`
- If you don't, the first tab (here, Basic HTTP) will be the initial selection instead
- You can use ordinary markdown syntax inside a `TabItem`
- **however**, due to bugs in our in-app markdown rendering library, you should be dilligent about using empty lines to separate different formatting-related things (surrounding tags and their contents, paragraphs vs lists, etc)
- You should also avoid indenting `TabItem` tags and their content according to html conventions, since text indented by four spaces (common for html nested inside two levels of tags) can be interpreted as a code block; different markdown rendering tools can handle this inconsistently.
```

Reply:
Good point! I started writing that bit as an example of using markdown syntax inside `Tabs`, and it only gradually evolved into useful documentation in its own right; but at this point, that's the main value add, and you're entirely right about where it should be.